test/login-user-test.ts

```diff
@@ -0,0 +1,43 @@
+import { describe, it } from "mocha";
+import { assert, expect } from "chai";
+import { url } from "../src/setup-server";
+import axios from "axios";
+
+describe("Hello Query Test", function () {
+  it("Tried a mock login attempt", async function () {
+    const response = await axios.post(url, {
+      query: `#graphql
+          mutation {
+            login(data: {
+              email: "felipe@example.com",
+              password: "123abc"
+            }) {
+              user {
+                id
+                name
+                email
+                birthDate
+              }
+              token
+            }
+          }
+        `
+    });
+
+    const expectedUser = {
+      id: "37",
+      name: "Test Login",
+      email: "login@email.com",
+      birthDate: "01-01-2000"
+    };
+
+    expect({
+      id: response.data.data.login.user.id,
+      name: response.data.data.login.user.name,
+      email: response.data.data.login.user.email,
+      birthDate: response.data.data.login.user.birthDate
```

Comment:
não dava pra fazer só um `expect(response.data.data.login.user).to.be.deep.eq(expectedUser)`? 
idealmente seria melhor colocar os objetos inteiros para pegar erros quando alterar a tipagem

Reply:
Verdade, dá pra fazer sim